src/components/BoardCard.js

```diff
@@ -11,24 +11,34 @@ import {
   fontSizes,
   shadows,
   transition,
+  fontWeights,
 } from '../theme'
+import { toAlpha } from '../utils/style'
 import ModalContainer from '../containers/ModalContainer'
 import UpdateBoardModal from './UpdateBoardModal'
 import DeleteBoardModal from './DeleteBoardModal'
+import Button from './Button'
+import { EllipsesIcon } from './Icon'
+import Flex from './Flex'
+import Spacer from './Spacer'
+import Dropdown, { MenuItem } from './Dropdown'
 
 const CardLink = glamorous(Link)({
-  padding: spacing[4],
+  padding: spacing[3],
   fontSize: fontSizes[3],
+  fontWeight: fontWeights.bold,
   textDecoration: 'none',
-  color: colors.gray[9],
+  color: toAlpha(colors.gray[9]),
   backgroundColor: colors.white,
   borderRadius: radii[1],
   boxShadow: shadows[1],
   outline: 0,
-  transition: `box-shadow ${transition.duration} ${transition.easing}`,
+  transitionProperty: 'box-shadow',
+  transitionDuration: transition.duration,
+  transitionTimingFunction: transition.easing,
 
   ':hover,:focus': {
-    boxShadow: shadows[3],
+    boxShadow: shadows[2],
```

Comment:
I think, as long as this is static on hover, `shadows[3]` looks a little better. But maybe it doesn't matter since we will be changing it anyways.